Subject: [PATCH] aufs-tools: don't strip executables

By default, aufs-tools strips its binaries. This produces QA ERROR
as follows:

   ERROR: QA Issue: File '/usr/lib/libau.so.2.7' from aufs-tools was already stripped, this will prevent future debugging! [already-stripped]

To prevent this, we remove -s in {Install} command in libau/Makefile.

diff --git a/libau/Makefile b/libau/Makefile
index 2fbd218..b0d6cba 100644
--- a/libau/Makefile
+++ b/libau/Makefile
@@ -45,7 +45,7 @@ install_ulib: File = ${LibSo}.${LibSoMajor}.${LibSoMinor}
 install_ulib: Tgt = ${DESTDIR}/usr/lib
 install_ulib: ${File}
 	install -d ${Tgt}
-	${Install} -m 644 -s ${File} ${Tgt}
+	${Install} -m 644  ${File} ${Tgt}
 	# -m 6755
 	ln -sf ${File} ${Tgt}/${LibSo}.${LibSoMajor}
 	ln -sf ${LibSo}.${LibSoMajor} ${Tgt}/${LibSo}
